plugins/Wallet/js/helpers.js

```diff
@@ -12,10 +12,12 @@ function convertSiacoin(hastings) {
 	// This deals with that imperfectly
 	var number = new BigNumber(hastings);
 	var ConversionFactor = new BigNumber(10).pow(24);
-	return number.dividedBy(ConversionFactor).round(2);
+
+	return number.gt(1) ? number.dividedBy(ConversionFactor).round(2)
+						 : number.dividedBy(ConversionFactor).toPrecision(1);
```

Comment:
Ah then this is the problem. `round()` returns a `BigNumber` and `toPrecision()` returns a `string`. This function should return one type of value.